templates/field/field--node--oe-documents--oe-tender--full.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * Theme override for the Documents field template of the Call for tenders content type.
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <div{{ item.attributes.addClass('ecl-u-mt-m') }}>{{ item.content }}</div>
```

Comment:
Didn't we have a feld--multiple template for this?

Reply:
feld--multiple is in EPIC-project branch.